package.json

```
@@ -57,6 +57,6 @@
     ]
   },
   "dependencies": {
-    "@types/node": "^12.11.1"
+    "@types/node": "*"
```

Thanks for the PR and the explanations. We support Node 8+, so I guess `>= 8` would be a better value?

```suggestion
    "@types/node": ">= 8"
```